src/dug/api.py

```diff
@@ -59,18 +59,18 @@ def __init__(self):
     def validate (self, request, component):
         return
         """ Validate a request against the schema. """
-        if not self.specs:
-            with open(schema_file_path, 'r') as file_obj:
-                self.specs = yaml.load(file_obj)
-        to_validate = self.specs["components"]["schemas"][component]
-        try:
-            app.logger.debug (f"--:Validating obj {json.dumps(request.json, indent=2)}")
-            app.logger.debug (f"  schema: {json.dumps(to_validate, indent=2)}")
-            jsonschema.validate(request.json, to_validate)
-        except jsonschema.exceptions.ValidationError as error:
-            app.logger.error (f"ERROR: {str(error)}")
-            traceback.print_exc (error)
-            abort(Response(str(error), 400))
+        # if not self.specs:
```

Comment:
why is it commented out?

Reply:
This is all due to issues with the JSON from elasticsearch not playing with the old flask api conversion to a dictionary. This is apparently a issue that is known but as of yet not solved however we do not need to worry about it since we have moved to FastAPI anyways